content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -22,6 +22,100 @@ Configuring the [aggregation layer](/docs/concepts/extend-kubernetes/api-extensi
+
+The high-level flow is as follows:
+
+1. apiserver: authenticate the requesting user and authorize their rights to the requested API path.
+2. apiserver: proxy the request to the extension server
+3. extension: authenticate the request from the apiserver
+4. extension: authorize the request from the original user
+5. extension: execute
+
+The rest of this section describes these steps in detail.
+
+### Apiserver Authentication and Authorization
+
+A request to an API path that is served by an extension server begins the same way as all API requests: communication to the apiserver. This path already has been registered with the apiserver by the extension server.
+
+The user communicates with the Kubernetes apiserver, requesting access to the path. The Kubernetes apiserver uses standard authentication and authorization configured with the apiserver to authenticate the user and authorize access to the specific path.
+
+For an overview of authenticating to a Kubernetes cluster, see ["Authenticating to a Cluster"](/docs/reference/access-authn-authz/authentication/). For an overview of authorization of access to Kubernetes cluster resources, see ["Authorization Overview"](/docs/reference/access-authn-authz/authorization/).
+
+Everything to this point has been standard Kubernetes API requests, authentication and authorization.
+
+The apiserver now is prepared to send the request to the extension server.
+
+### Apiserver Proxy the Request
+
+The apiserver now will send, or proxy, the request to the extension server that registered to handle the request. In order to do so, it needs to know several things:
+
+1. How should the apiserver authenticate to the extension server, informing the extension server that the request, which comes over the network, is coming from a valid apiserver?
+2. How should the apiserver inform the extension server of the username and group for which the original request was authenticated?
+
+In order to provide for these two, you must configure the apiserver using several flags.
+
+#### Apiserver Client Authentication
+
+The Kubernetes apiserver authenticates to the extension server using a client certificate. You must provide the following to the Kubernetes apiserver upon startup, using the provided flags:
+
+* private key file via `--proxy-client-key-file`
+* signed client certificate file via `--proxy-client-cert-file`
+* certificate of the CA that signed the client certificate file via `--requestheader-client-ca-file`
+* valid Common Names (CN) in the signed client certificate via `--requestheader-allowed-names`
+
+The Kubernetes apiserver will use the files indicated by `--proxy-client-*-file` to authenticate to the extension server. In order for the request to be considered valid by a compliant extension server, the following conditions must be met:
+
+1. The connection must be made using a client certificate that is signed by the CA whose certificate is in `--requestedheader-client-ca-file`.
```

Comment:
typo: requestheader

Reply:
Nah, it requested it, in the past. LOL!

Fixed.